catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/GeoPdfParserImpl.java

```diff
@@ -113,7 +115,12 @@ public String apply(PDDocument pdfDocument) throws IOException {
             if (neatlineArray == null) {
               neatlineArray = generateNeatLineFromPDFDimensions(pdPage);
             }
-            polygons.add(getWktFromNeatLine(lgidict, neatlineArray, toDoubleVisitor));
+
+            Optional<String> wkt = getWktFromNeatLine(lgidict, neatlineArray, toDoubleVisitor);
+            if (wkt.isPresent()) {
```

Comment:
❗️ Use `ifPresent()` and call add in the lambda instead of calling `ifPresent()` followed by `get()`

Reply:
`wtk.ifPresent(w-> polygons.add(w))`